### DISCUSSION TITLE
Add Package Website to Sidebar

### DISCUSSION THREAD

**Discussion by Sherlouk** [SwiftPackageIndex/SwiftPackageIndex-Server#1686] [Ideas]
Many repositories add a website to their GitHub repository which often links to documentation or an overview of functionality. 

Some random matching examples:
- https://github.com/scinfu/SwiftSoup
- https://github.com/d-exclaimation/pioneer
- https://github.com/tuist/tuist

It would be nice if this URL was added to the sidebar on the package page.

![Captured 2022-04-11 at 1 07 16](https://user-images.githubusercontent.com/15193942/162736088-581a83c2-ad0c-42b0-b7d6-62beb3a15d90.png)

**Comment by daveverwer** [comment]
I presume this comes out as a separate field from the API? So, we'd need to modify ingest, and get it into the `repositories` table, bearing it through into the view model and out to the page.

We've just had a clean up of that right-hand side menu, so I might try and find another place for it, but I'm onboard with this. Are you interested in implementing it? If so, I'll promote this to an issue,.

**Comment by Sherlouk** [comment]
Aye I believe it is a new field in the API, so would need to update the GraphQL query.

If you can provide tips on how you want it to look and the link text etc then I'm happy to work on this 👍

**Reply by daveverwer** [reply]
Yea absolutely. Just stick it anywhere for now and we can have a play with where it might go once it's pulled through into the view. 👍
